includes/Classifai/Providers/OpenAI/TextToSpeech.php

```diff
@@ -0,0 +1,323 @@
+<?php
+/**
+ * OpenAI Text to Speech integration
+ */
+
+namespace Classifai\Providers\OpenAI;
+
+use Classifai\Providers\Provider;
+use Classifai\Features\TextToSpeech as FeatureTextToSpeech;
+use WP_Error;
+
+class TextToSpeech extends Provider {
+	use OpenAI;
+
+	const ID = 'openai_text_to_speech';
+
+	/**
+	 * OpenAI Text to Speech URL.
+	 *
+	 * @var string
+	 */
+	protected $api_url = 'https://api.openai.com/v1/audio/speech';
+
+	/**
+	 * OpenAI TextToSpeech constructor.
+	 *
+	 * @param \Classifai\Features\Feature $feature_instance The feature instance.
+	 */
+	public function __construct( $feature_instance = null ) {
+		$this->feature_instance = $feature_instance;
+	}
+
+	/**
+	 * Register settings for the provider.
+	 */
+	public function render_provider_fields(): void {
+		$settings = $this->feature_instance->get_settings( static::ID );
+
+		add_settings_field(
+			static::ID . '_api_key',
+			esc_html__( 'API Key', 'classifai' ),
+			[ $this->feature_instance, 'render_input' ],
+			$this->feature_instance->get_option_name(),
+			$this->feature_instance->get_option_name() . '_section',
+			[
+				'option_index'  => static::ID,
+				'label_for'     => 'api_key',
+				'input_type'    => 'password',
+				'default_value' => $settings['api_key'],
+				'class'         => 'classifai-provider-field hidden provider-scope-' . static::ID,
+				'description'   => sprintf(
+					wp_kses(
+						/* translators: %1$s is replaced with the OpenAI sign up URL */
+						__( 'Don\'t have an OpenAI account yet? <a title="Sign up for an OpenAI account" href="%1$s">Sign up for one</a> in order to get your API key.', 'classifai' ),
+						[
+							'a' => [
+								'href'  => [],
+								'title' => [],
+							],
+						]
+					),
+					esc_url( 'https://platform.openai.com/signup' )
+				),
+			]
+		);
+
+		add_settings_field(
+			static::ID . '_tts_model',
+			esc_html__( 'TTS model', 'classifai' ),
+			[ $this->feature_instance, 'render_select' ],
+			$this->feature_instance->get_option_name(),
+			$this->feature_instance->get_option_name() . '_section',
+			[
+				'option_index'  => static::ID,
+				'label_for'     => 'tts_model',
+				'options'       => [
+					'tts-1'    => __( 'Text-to-speech 1 (Optimized for speed)', 'classifai' ),
+					'tts-1-hd' => __( 'Text-to-speech 1 HD (Optimized for quality)', 'classifai' ),
+				],
+				'default_value' => $settings['tts_model'],
+				'description'   => __( 'Select a model depending on your requirements.', 'classifai' ),
```

Comment:
Might be nice to link out to the [documentation](https://platform.openai.com/docs/models/tts) here so people can better determine the differences between the models.

Maybe just link the word `model` in that sentence to the docs?